directory/datadog_checks/directory/config.py

```diff
@@ -0,0 +1,33 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from os.path import abspath
+from re import compile as re_compile
+
+from datadog_checks.base import ConfigurationError, is_affirmative
+
+MAX_FILEGAUGE_COUNT = 20
+
+
+class DirectoryConfig(object):
+    def __init__(self, instance):
+        try:
+            directory = instance['directory']
+        except KeyError:
+            raise ConfigurationError('DirectoryCheck: missing `directory` in config')
```

Comment:
```suggestion
        if not 'directory' in instance:
            raise ConfigurationError('DirectoryCheck: missing `directory` in config')
        directory = instance['directory']
```

Reply:
```suggestion
        if 'directory' not in instance:
            raise ConfigurationError('DirectoryCheck: missing `directory` in config')
        directory = instance['directory']
```

Reply:
@hithwen Could you review again, I just accept the change (slightly modified). And also added a test for it.